packages/shared/routes/login/views/EnterPin.svelte

```diff
@@ -172,10 +172,19 @@
                 strongholdVersion={$activeProfile?.strongholdVersion ?? -1}
                 bgColor="blue"
             />
+            {#if isStrongholdOutdated($activeProfile?.strongholdVersion)}
+                <TextHint
```

Comment:
needs some retouch for darkmode 🙏🏼 
![image](https://user-images.githubusercontent.com/3624944/226642424-85bbdbc8-2e08-4ea1-bd9d-188b1e7d8e89.png)